netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -165,4 +168,18 @@ private static ApplicationProtocolConfig selectApplicationProtocolConfig(SslProv
         throw new IllegalArgumentException("Unsupported provider: " + provider);
     }
   }
+
+  static void ensureAlpnAndH2Enabled(ApplicationProtocolNegotiator alpnNegotiator) {
+    checkArgument(alpnNegotiator != null, "ALPN must be configured");
+    checkArgument(alpnNegotiator.protocols() != null && !alpnNegotiator.protocols().isEmpty(),
+        "ALPN must be enabled and list HTTP/2 as a supported protocol.");
+    for (String protocol : alpnNegotiator.protocols()) {
```

Comment:
As an option, we could go ahead and get rid of the list of HTTP2 versions and only have one. Then we could use protocols().contains().